src/runtime_src/core/tools/xbutil2/SubCmdExamine.cpp

```diff
@@ -114,7 +114,7 @@ SubCmdExamine::execute(const SubCmdOptions& _options) const
   commonOptions.add_options()
     ("device,d", boost::program_options::value<decltype(devices)>(&devices)->multitoken(), "The Bus:Device.Function (e.g., 0000:d8:00.0) device of interest.  A value of 'all' (default) indicates that every found device should be examined.")
     ("report,r", boost::program_options::value<decltype(reportNames)>(&reportNames)->multitoken(), (std::string("The type of report to be produced. Reports currently available are:\n") + reportOptionValues).c_str() )
-    ("format,f", boost::program_options::value<decltype(sFormat)>(&sFormat), (std::string("Report output format. Valid values are:\n") + formatOptionValues).c_str() )
+    ("format,f", boost::program_options::value<decltype(sFormat)>(&sFormat)->default_value("json"), (std::string("Report output format. Valid values are:\n") + formatOptionValues).c_str() )
```

Comment:
Hmm...this doesn't look right.  Why is there a default value here?

See comment below for further detail.